src/GTranslate/LanguageDictionary.cs

```diff
@@ -142,153 +142,167 @@ public bool TryGetLanguage(string code, [MaybeNullWhen(false)] out Language lang
         ["ar"] = new("Arabic", "العربية", "ar", "ara"),
-        ["hy"] = new("Armenian", "Հայերեն", "hy", "hye"),
+        ["as"] = new("Assamese", "অসমীয়া", "as", "asm", TranslationServices.Bing | TranslationServices.Microsoft),
+        ["ay"] = new("Aymara", "aymar aru [ajmaɾ ˈaɾu]", "ay", "aym", TranslationServices.Google),
         ["az"] = new("Azerbaijani", "Azərbaycan", "az", "aze"),
-        ["eu"] = new("Basque", "Euskara", "eu", "eus"),
+        ["ba"] = new("Bashkir", "Bashkir", "ba", "bak", TranslationServices.Bing | TranslationServices.Microsoft | TranslationServices.Yandex),
         ["be"] = new("Belarusian", "беларуская", "be", "bel", TranslationServices.Google | TranslationServices.Yandex),
+        ["bg"] = new("Bulgarian", "Български", "bg", "bul"),
+        ["bho"] = new("Bhojpuri", "भोजपुरी", "bho", "bho", TranslationServices.Google),
+        ["bm"] = new("Bambara", "ߓߊߡߊߣߊ߲ߞߊ߲", "bm", "bam", TranslationServices.Google),
         ["bn"] = new("Bengali", "বাংলা", "bn", "ben"),
+        ["bo"] = new("Tibetan", "བོད་སྐད་", "bo", "bod", TranslationServices.Bing | TranslationServices.Microsoft),
         ["bs"] = new("Bosnian", "bosanski", "bs", "bos"),
-        ["bg"] = new("Bulgarian", "Български", "bg", "bul"),
-        ["my"] = new("Burmese", "မြန်မာ", "my", "mya"),
         ["ca"] = new("Catalan", "Català", "ca", "cat"),
         ["ceb"] = new("Cebuano", "Binisaya", "ceb", "ceb", TranslationServices.Google | TranslationServices.Yandex),
-        ["ny"] = new("Chichewa", "Nyanja", "ny", "nya", TranslationServices.Google),
-        ["zh-CN"] = new("Chinese (Simplified)", "中文 (简体)", "zh-CN", "zho-CN"),
-        ["zh-TW"] = new("Chinese (Traditional)", "繁體中文 (繁體)", "zh-TW", "zho-TW", TranslationServices.Google | TranslationServices.Bing | TranslationServices.Microsoft),
         ["co"] = new("Corsican", "Corsu", "co", "cos", TranslationServices.Google),
-        ["hr"] = new("Croatian", "Hrvatski", "hr", "hrv"),
         ["cs"] = new("Czech", "Čeština", "cs", "ces"),
+        ["cv"] = new("Chuvash", "Чӑвашла", "cv", "chv", TranslationServices.Yandex),
+        ["cy"] = new("Welsh", "Cymraeg", "cy", "cym"),
         ["da"] = new("Danish", "Dansk", "da", "dan"),
-        ["nl"] = new("Dutch", "Nederlands", "nl", "nld"),
+        ["de"] = new("German", "Deutsch", "de", "deu"),
+        ["dgo"] = new("Dogri", "डोगरी (ḍogrī) ڈوگرى", "dgo", "dgo", TranslationServices.Google),
+        ["dv"] = new("Divehi", "ދިވެހިބަސް", "dv", "div", TranslationServices.Bing | TranslationServices.Microsoft),
+        ["ee"] = new("Ewe", "Eʋegbe", "ee", "ewe", TranslationServices.Google),
+        ["el"] = new("Greek", "Ελληνικά", "el", "ell"),
+        ["emj"] = new("Emoji", "Emoji", "emj", "emj", TranslationServices.Yandex), // Not present in Yandex.Cloud
         ["en"] = new("English", "English", "en", "eng"),
+        ["en-GB"] = new("English (UK)", "English (UK)", "en-GB", "en-GB", TranslationServices.Google),
```

Comment:
As mentioned in the comments, this should be removed because the API treats `en` and `en-GB` as the same language.
```suggestion
```

Reply:
Here, it does seem that en-GB is ignored, and all the translations are coming back in en-US.
I tried entering a British sentence in the online [Google ](https://translate.google.com/?sl=auto&tl=en&text=The%20honour%20of%20the%20barrister%20was%20attacked%20by%20his%20neighbour%20who%20had%20a%20biscuit%20in%20the%20boot%20of%20his%20car%20due%20to%20his%20good%20humour%20and%20hard%20labour.%20&op=translate) and [Bing  ](https://www.bing.com/translator) translator, and both brought back the exact same sentence.

> The honour of the barrister was attacked by his neighbour who had a biscuit in the boot of his car due to his good humour and hard labour. 

The above sentence should have come back to the following:
>The honor of the attorney was attacked by his neighbor who had a cookie in the trunk of his car due to his good humor and hard labor. 

Even the following link, only changed four words (honour, neighbour, humour, & labour).
https://gotranscript.com/translation-services/british-to-american#text

It doesn't look like there is a good translation service for en-GB.

I've removed it.